packages/swap/src/routes/__tests__/__snapshots__/quote.test.ts.snap

```diff
@@ -40,6 +40,90 @@ exports[`server GET /quote gets the quote for deprecated params without the chai
 }
 `;
 
+exports[`server GET /quote gets the quote from btc with boost information 1`] = `
+{
+  "boostedQuote": {
+    "boostFeeBps": 5,
```

Comment:
should we call this `effectiveBoostFeeBps`?

Reply:
hmm, i do agree that `boostFeeBps` might not be very clear and it helps being more explicit here but calling it `effective` might also be wrong as it is only an estimate at this point. What about `boostFeeBpsEstimate` or `estimatedBoostFeeBps` ?

Reply:
Used `estimatedBoostFeeBps`